events/models.py

```diff
@@ -3,31 +3,48 @@
 
 
 class Event(models.Model):
-    event = models.CharField(max_length=100, blank=True)
-    description = models.CharField(max_length=1000, blank=True)
-    start_time = models.DateTimeField()
-    end_time = models.DateTimeField()
+    priority = models.IntegerField(blank=True)
+    event = models.CharField(max_length=100, blank=False)
+    description = models.CharField(max_length=2000, blank=True)
     prize = models.CharField(max_length=20, blank=True)
-    venue = models.CharField(max_length=10, blank=True)
-    image = models.ImageField(upload_to='event-images/', null=True, blank=True)
+    team_size = models.CharField(max_length=20, blank=True)
+    venue = models.CharField(max_length=50, blank=True)
+    start_time = models.CharField(max_length=100, blank=False)
+    end_time = models.CharField(max_length=100, blank=False)
```

Comment:
Can we use `DateTimeField` [if frontend can send valid strings] ?

Reply:
Previously, I used DateTimeField but the format was also including UTC+5:30 without spaces in the string so it didn't match the design format sir